.travis.yml

```diff
@@ -116,7 +116,16 @@ before_install:
       sudo apt-get update;
       sudo apt-get install sagemath-upstream-binary;
     fi
+#The install cycle below is to test installation on systems without setuptools.
 install:
+  - if [[ "${TEST_AUTOWRAP}" == "true" ]]; then
+      pip uninstall -y setuptools;
+      sudo apt-get remove --purge python-setuptool;
```

Comment:
> Also, we need to make sure that after you call pip uninstall -y setuptools a second one isn't installed in the apt-get installs

I added ` sudo apt-get remove --purge python-setuptools;` after reading this.
Should I keep it ? It would cause no harm anyway.